From cc0977eacfd1975ba0880027c3fe2416b25e0256 Mon Sep 17 00:00:00 2001
From: Tam Nguyen <tam.nguyen.xa@renesas.com>
Date: Tue, 27 May 2025 21:51:28 +0700
Subject: [PATCH 30/46] arm64: dts: renesas: r8a78000: Add device tree node for
 Ethernet PCS

Add the device tree node for the Ethernet PCS (Physical Coding Sublayer)
controller on the R-Car X5H (R8A78000) SoC.

Signed-off-by: Tam Nguyen <tam.nguyen.xa@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a78000.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a78000.dtsi b/arch/arm64/boot/dts/renesas/r8a78000.dtsi
index 6a4d54dcf8db..dbc08ba7ced9 100644
--- a/arch/arm64/boot/dts/renesas/r8a78000.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a78000.dtsi
@@ -933,6 +933,13 @@ mp_phy: mp_phy@c9a00000 {
 			#phy-cells = <2>;
 			status = "disabled";
 		};
+
+		eth_pcs: phy@c9c50000 {
+			compatible = "renesas,r8a78000-ether-pcs";
+			reg = <0 0xc9c50000 0 0x4000>;
+			#phy-cells = <1>;
+			status = "disabled";
+		};
 	};
 
 	timer {
-- 
2.34.1

